apps/weshipit/components/french-react-native-apps/format-apps-by-category.tsx

```diff
@@ -0,0 +1,28 @@
+function extractAppData(app) {
```

Comment:
1. rename fiel to `.ts` :pray
2. create a `.fixture.ts` file with example data
3. add a unit test file